schemas/_common-schema.json

```diff
@@ -45,7 +45,7 @@
         "type": {
           "type": "string",
           "description": "Type of the session",
-          "enum": ["user", "synthetics"],
+          "enum": ["user", "synthetics", "ci-test"],
```

Comment:
what about having the same wording for both session type and extra context key?
`ci-visibility` VS `ci-test`